docs/data/pages.ts

```diff
@@ -109,39 +109,49 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
           { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
           { pathname: '/x/react-data-grid/aggregation', title: 'Aggregation', plan: 'premium' },
-          { pathname: '/x/react-data-grid/pivoting', title: 'Pivoting 🚧', plan: 'premium' },
+          {
+            pathname: '/x/react-data-grid/pivoting',
+            title: 'Pivoting',
+            plan: 'premium',
+            planned: true,
+          },
         ],
       },
       {
         pathname: '/x/react-data-grid/server-side-data-group',
-        title: 'Server-side data 🚧',
-        plan: 'pro',
+        title: 'Server-side data',
+        planned: true,
```

Comment:
Cool, is this a new flag? Should we also update it for other packages like charts which still have 🚧 for the components that are upcoming, or does it mean something specific?